superset-frontend/src/views/CRUD/types.ts

```diff
@@ -45,6 +46,19 @@ export interface Dashboard {
   loading?: boolean;
 }
 
+export interface DashboardCardProps {
```

Comment:
I moved this into the DashboardCard component file in https://github.com/apache/incubator-superset/pull/11502 since I thought it made sense to keep it next to the component. Can we keep it there? It doesn't appear we're importing this anywhere else.

Reply:
Will do that makes sense!